numpy/typing/__init__.py

```diff
@@ -68,13 +68,27 @@
 
 .. code-block:: python
 
-    x = np.array([1, 2])
-    x.dtype = np.bool_
+    >>> x = np.array([1, 2])
+    >>> x.dtype = np.bool_
 
 This sort of mutation is not allowed by the types. Users who want to
 write statically typed code should insted use the `numpy.ndarray.view`
 method to create a view of the array with a different dtype.
 
+dtype
+~~~~~
+
+The ``_DTypeLike`` static type tries to avoid creation of dtype objects using
```

Comment:
Why `_DTypeLike` when it's `DTypeLike` (i.e. public)?

Reply:
Fixed.